src/components/App.jsx

```diff
@@ -1,13 +1,39 @@
-import React, {Fragment} from "react";
-import Example from '../containers/exampleCounterContainer.js';
+import React, { Fragment } from "react";
+import Example from "../containers/exampleCounterContainer.js";
 
-const App = ({name}) => {
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+
+import LandingPage from "./MainComponents/LandingPage_Components/LandingPage.jsx";
```

Comment:
I am going to accept the other pull requests first so you can change these to be Christina and Anton's components! Will let you know when I have done this :)